common/app/model/content.scala

```diff
@@ -233,7 +237,52 @@ object Content {
     }
   }
 
-  def make(apiContent: contentapi.Content): Content = {
+  object BodyBlocks {
+    def make(blocks: Option[Blocks]): Seq[BodyBlocks] =
+    blocks.toSeq.flatMap { blocks =>
```

Comment:
true, I decided to open my mind a bit to the debate to see what the rest of the internet says, and there's not much.  The best I could find was this: http://stackoverflow.com/questions/7031106/why-does-scala-support-shadow-variables